### PR TITLE
run benchmark with mechx

### PR DESCRIPTION
**Pull request by richardblythman**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,3 +4,6 @@
 [submodule "benchmark/mech"]
 	path = benchmark/mech
 	url = https://github.com/valory-xyz/mech.git
+[submodule "benchmark/mech-client"]
+	path = benchmark/mech-client
```
Comment (DavidMinarsch) [inline .gitmodules:8]: Why do you not simply install from pypi?

File: benchmark/pyproject.toml [hunk 2]
```diff
@@ -15,12 +15,14 @@ ruff = "^0.1.13"
 anthropic = "^0.3.11"
 tiktoken = "^0.5.1"
 huggingface-hub = "^0.20.3"
-mech = { path = "./mech", develop = true }
+# mech = { path = "./mech", develop = true }
+mech_client = { path = "./mech_client", develop = true }
```
Comment (DavidMinarsch) [inline benchmark/pyproject.toml:19]: I suggest you install from pypi instead?

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

**Comment by richardblythman** [comment]
With pip package I'm getting:
`Error running benchmark for tool prediction-offline: interact() got an unexpected keyword argument 'extra_attributes'`

Was working with the submodule. Do we need to publish the latest version of mech-client on pypi? @DavidMinarsch

**Comment by richardblythman** [comment]
It looks like the main open PRs have been merged. Can you confirm @0xArdi

**Pull request merged**